src/lib/Stepper/Step.svelte

```diff
@@ -33,11 +33,6 @@
     display: flex;
     align-items: center;
   }
-  .step-vertical {
```

Comment:
This should be left in there. Has no effect on the stepper itself. Otherwise the orientation prop should be removed if we are not going to allow a vertical orientation with the step.